connect/src/routes/tokenBridgeRoute.ts

```diff
@@ -0,0 +1,87 @@
+import { Network } from "@wormhole-foundation/sdk-base";
+import {
+  Signer,
+  TokenTransferDetails,
+  TransactionId,
+  isSameToken,
+} from "@wormhole-foundation/sdk-definitions";
+import { Wormhole } from "../wormhole";
+import { Route, TransferRequest, ValidationError, ValidationResult } from "./route";
+import { TokenTransfer } from "../protocols/tokenTransfer";
+
+export type TokenBridgeTransferOptions = {
+  payload?: Uint8Array;
+};
+
+export class TokenBridgeRoute<N extends Network> extends Route<N, TokenBridgeTransferOptions> {
+  NATIVE_GAS_DROPOFF_SUPPORTED = false;
+  IS_AUTOMATIC = false;
+
+  constructor(wh: Wormhole<N>, request: TransferRequest) {
```

Comment:
I think you could just not have this function here, and `super` will take it automatically